workflow_tests/createNewVersion.test.ts

```diff
@@ -1,12 +1,13 @@
-const path = require('path');
-const kieMockGithub = require('@kie/mock-github');
-const utils = require('./utils/utils');
-const assertions = require('./assertions/createNewVersionAssertions');
-const mocks = require('./mocks/createNewVersionMocks');
-const ExtendedAct = require('./utils/ExtendedAct').default;
+import {MockGithub} from '@kie/mock-github';
+import path from 'path';
+import assertions from './assertions/createNewVersionAssertions';
+import * as mocks from './mocks/createNewVersionMocks';
```

Comment:
```suggestion
import mocks from './mocks/createNewVersionMocks';
```

Merge with `main` first